src/controllers/userController.js

```diff
@@ -68,6 +71,62 @@ export default class UserController {
     }
   }
 
+  static Providelink(req, res) {
+    const { email } = req.body;
+    models.users.findOne({
+      where: {
+        email,
+      },
+    }).then(user => {
+      if (user == null) {
+        return responseUtil(res, 404, strings.users.error.USER_NOT_FOUND);
+      }
+      const verifyToken = EmailToken.ResetToken(user);
+      sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+      const msg = {
+        to: email,
+        from: 'BarefootNomad@gmail.com',
+        subject: 'Barefoot Nomad Resetpassword Link',
+        html: `<div style="font-family:Avenir,Helvetica,sans-serif;box-sizing:border-box;padding:35px;">
+        <h1 style="color: #848484;">Barefoot Nomad</h1>
+        <p style="font-family:Avenir,Helvetica,sans-serif;box-sizing:border-box;color:#74787e;font-size:16px;line-height:1.5em;margin-top:0;text-align:left">Welcome Back,<br> We are happy to be with you. Please you can reset your password now.<br> Click the button below to reset your password.</p>
+        <p><a style="background-color: #3097d1; border: 2px solid #3097d1; padding: 8px; color: #fff; font-size: 16px; text-decoration: none;cursor: pointer;" href="${req.protocol}://${req.headers.host}/api/v1/users/resetpassword/${verifyToken}">Reset passowrd</a>
```

Comment:
I am still not convinced by you sending a link that tells me **404, route not found**. As a user what do I do when I get this? I think it is broken. Reach out to @zellymiriam  as soon as possible @sabin18, if she agrees with that implementation, but I am not convinced